src/test/java/com/google/jenkins/GoogleOAuthPlugInTestSuite.java

```diff
@@ -0,0 +1,10 @@
+package com.google.jenkins;
+
+import com.google.jenkins.plugins.CredentialsOAuthTestSuite;
+import com.google.jenkins.plugins.UtilTestSuite;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses(value = {CredentialsOAuthTestSuite.class, UtilTestSuite.class})
+public class GoogleOAuthPlugInTestSuite {}
```

Comment:
s/GoogleOAuthPlugInTestSuite/GoogleOAuthPluginTestSuite

Reply:
To clarify, could you please rename this class to: GoogleOAuthPluginTestSuite. Thanks.

Reply:
Done